src/components/LoanCollections/HomeExp/KivaLoanCardCarousel.vue

```diff
@@ -0,0 +1,171 @@
+<template>
```

Comment:
@edwinf-barreram Did you explore making this loan card type a prop on the existing loan card carousel we use on homepage? That would be my preferred approach rather than having an entirely different component with a lot of duplicated code. It would also allow us to re-use the setup coming from contentful and just add that config to the dataObject there...

What I've described above will be the process for when we port this to cms-page-server

Reply:
@mcstover I tried it initially but due to the quantity of customizations that I was having to add there just to show the different versions and due cards have different sizes and the new carousel version must show 4 cards instead of 3, it was making it difficult to take control of everything. It was the reason why I decided to create a new component in a different folder just for the experiment.

Also... thinking on future when we have to delete either of them it don't be a pain in the ass. 😅

Reply:
Ok thanks for considering this. I'll likely still setup a switch to choose a loan card in the new app.